### ISSUE TITLE
Question: Can I use the same set of keys for move_down/up and move_visual_mark_down/up?

### ISSUE THREAD

**Issue by AmaiKinono** [ahrm/sioyek#207]
I hope when visual mark is not enabled, I can use `j/k` to move up/down; and when it's enabled, I can use `j/k` to move the visual mark.

I do this in `keys_user.config`:

```
move_down           j
move_up             k
...
move_visual_mark_up	k
move_visual_mark_down	j
```

But then `j/k` only work as `move_visual_mark_up/down`.

**Comment by ahrm** [comment]
Fixed in 99109b8f7281e27d325effba31dd81e046a1f3cd. Now `move_visual_mark_` behaves like `move_*` when not in visual mark mode.

**Comment by AmaiKinono** [comment]
Works like a charm. Thanks!
